### PR TITLE
WIP - basic skeleton

### PR DESCRIPTION
**Pull request by alexwilson**

### CODE REVIEW

File: _events/2018-03-27-railway-oriented-programming.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-# Todo Generate the files in _events using the meetup api and a scheduled task that rebuilds the repo
 title: Railway-Oriented Programming with Stuart Herbert
-date: Thu Feb 22 2018
+date: Thu Mar 27 2018
 description: >
     Another warm welcome to brumPHP! We're building on our successful introductory event by welcoming veteran PHP speaker Stuart Herbert to 1000 Trades in the Jewellery Quarter, to talk about functional programming
 link: https://www.meetup.com/brumphp/events/246966799/
 location: Talis
```
Comment (SecondeJK) [inline _events/2018-03-27-railway-oriented-programming.md]: Location isn't Talis, it's 1000 Trades y'all

**Comment by aceew** [comment]
Hey, so right now we still need a background image for the jumbotron, but the group photos I have are portrait photos. Any objection to merging this now to have something up and then updating after I've got one tomorrow?

We'll also need to add the code of conduct back in when we have one, might be worth creating issues for things we still need to do.  

Also do we have a domain? If so I can set up the cloudflare cname for this.

Open to ideas.
cc: @antoligy @SecondeJK

**Comment by SecondeJK** [comment]
I suggested something about the Jumbo background but I can't for the life of me remember what it is. We don't have a domain, we need one. Can you set up access for cloudflare, I can buy the domain etc.?

**Comment by SecondeJK** [comment]
If ok with @antoligy who will probably be waking up round about now, I suggest merge

**Comment by SecondeJK** [comment]
Let's do this

**Pull request merged**